api/v1/_submissions/PKPBackendSubmissionsHandler.inc.php

```diff
@@ -132,9 +132,10 @@ public function getSubmissions($slimRequest, $response, $args) {
 				case 'orderDirection':
 					$params[$param] = $val === 'ASC' ? $val : 'DESC';
 					break;
-
+				case 'daysInactive':
+					$params[$param] = (int) $val[0];
```

Comment:
I _think_ this should be `$val` (without the `[0]`).

Reply:
If I'm not wrong, because of 
https://github.com/pkp/ui-library/blob/0e3df76b57b7aedae3e65cc30fc8ccc4a77e5803/src/components/ListPanel/submissions/SubmissionsListFilter.vue#L70-L74
arguments for filters are arrays (and that's the case)

Reply:
Ah yes, the new methods for filter setting booleans are only available in `master`. We'll need to do some additional work to port this to `master`. 

Let's leave it as-is for stable and we'll make it an undocumented param so that people don't start using it before we change things in `master`.